client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -267,19 +267,25 @@ define([
 
     var optimizationTimer;
 
-    $scope.startOptimization = function () {
-      $http.post('/api/analysis/optimization/start', $scope.params)
-        .success(function(data, status, headers, config) {
-          if (data.status == "OK" && data.join) {
-      // Keep polling for updated values after every 5 seconds till we get an error.
-      // Error indicates that the model is not calibrating anymore.
-            optimizationTimer = $interval(checkWorkingOptimization, 5000, 0, false);
-            $scope.optimizationStatus = statusEnum.RUNNING;
-          } else {
-            console.log("Cannot poll for optimization now");
+      $scope.startOptimization = function () {
+          if($scope.yearError === true) {
```

Comment:
@kitze Is not clear to me where the yearError is eventually evaluated, looks like noone. Am I missing anything?